armi/settings/setting.py

```diff
@@ -115,6 +115,10 @@ def __init__(
         """
         self.name = name
         self.description = description or name
+        runLog.warning(
+            f"DeprecationWarning: Setting {name} defined without description.",
+            single=True,
+        )
```

Comment:
Isn't this deprecation warning going to be printed all the time...? Like, shouldn't it be wrapped under an if-statement?

Reply:
No, this is why I have `single=True`. That means that this warning will only ever be printed ONCE during a run.

Reply:
Sorry, let me clarify. This warning will be printed indiscriminately. There is no check of if the setting actually has a description or not. The runLog statement is hit even if the setting has a description.

Reply:
Fixed